packages/mdc-dialog/README.md

```diff
@@ -204,7 +204,7 @@ dialog.listen('MDCDialog:cancel', function() {
   console.log('canceled');
 })
 
-document.querySelector('#default-dialog-activation').addEventListener('click', function (evt) {
+document.querySelector('#show-dialog-button').addEventListener('click', function (evt) {
```

Comment:
why did you make this change?

Reply:
I just did a search-and-replace in the whole project and didn't catch this